source/configuring-ember/specifying-url-type.md

```diff
@@ -13,30 +21,22 @@ Router.map(function() {
 });
 ```
 
+## history
```

Comment:
:+1: `history`, `hash`, `auto` (because we already both the previous two), `none`?